pyrasterframes/src/main/python/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+
+global-exclude *.py[cod] __pycache__ .DS_Store
+recursive-include deps/jars *.jar
+
```

Comment:
I think this needs  
```suggestion
include requirements.txt
```

Reply:
According to [this](https://stackoverflow.com/questions/24727709/do-python-projects-need-a-manifest-in-and-what-should-be-in-it), the `MANIFEST.in` file is superfluous.

Reply:
I believe the [comment](https://stackoverflow.com/questions/24727709/do-python-projects-need-a-manifest-in-and-what-should-be-in-it#comment46482024_24727824) that touches on use of MANIFEST.in for [package-data](https://github.com/locationtech/rasterframes/pull/144/files#diff-ed7242f683fdbaa1990121cfc7383c95R159) is the most relevant.

Reply:
Or I was off base. I think the need for `requirements.txt` comes from the pattern where we are trying to `open` requirements.txt in the setup.py. In the pip installed enviornment that file is missing unless in MANIFEST.

I have another discussion about possibly removing that `open` from setup.py.

Reply:
Also know that the `requirements.txt` can be used by PyCharm.
For background, I moved things into the `requirements.txt` so we had a single point of definition, and there seemed to be `setuptools` stuff that wanted it (?).

Reply:
OK, I see where you're going below.